content/blog/2020/07/2020-07-29-git-performance-improvement-phase2.adoc

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Git Plugin Performance Improvement Phase-2 Progress"
+tags:
+- plugins
+- git
+- platform-sig
+- developer
+- performance
+- gsoc
+- gsoc2020
+author: rishabhbudhouliya
+opengraph:
+  image: /images/post-images/2020/07-git-plugin-performance-improvement/gsoc-git-opengraph.png
+---
+
+The second phase of the link:https://www.jenkins.io/blog/2020/07/09/git-performance-improvement-phase1/[Git Plugin Performance Improvement project] has been great in terms of the progress we have achieved in implementing performance improvement insights
+derived from the phase-1 link:https://openjdk.java.net/projects/code-tools/jmh/[JMH] micro-benchmark experiments.
+
+As we all know that performance of a git fetch is highly correlated to the size of the remote repository, our task was to find the difference in performance for the two available git implementations in the link:https://github.com/jenkinsci/git-plugin[Git Plugin], "Git" and "JGit".
+
+Our major finding was that "Git" performs much better than "JGit" when it comes to a large sized repository (>100 MiB). Interestingly, "JGit" performs better than "Git" when size of the repository is less than 100 MiB. 
```

Comment:
I wonder if markup would work for the post. 

For example: `git` instead of "Git" and `JGit` instead of "JGit"

Reply:
I've added backticks to format `git` and `Jgit` consistently across the blog.